test/powershell/engine/Basic/RegisterAssemblyResolverEarly.ps1

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+Describe "Assembly resolvers should be registered early at startup" -Tags "CI" {
+
+    ## The PKI module requires loading an assembly from GAC, and thus depends on the PowerShell assembly resolver to work.
+    It "Can load the PKI module with 'pwsh -ExecutionPolicy bypass -NoProfile -c `"Import-Module PKI`"'" -Skip:(!$IsWindows) {
+        $pwsh = Join-Path $PSHOME "pwsh.exe"
+        $out = & $pwsh -ExecutionPolicy bypass -NoProfile -c "Import-Module PKI; Get-Module | % name"
```

Comment:
why is executionpolicy needed here?

Reply:
> why is executionpolicy needed here?

Because checking if a file was acceptable for the current execution policy was triggering `ClrFacade` initialization. Bypass skips the check so is needed for the test

Reply:
I would be good to have a comment for this

Reply:
A comment is added.